docs/mongocli/command/mongocli-atlas-quickstart.txt

```diff
@@ -89,6 +89,10 @@ Options
      - 
      - false
      - Indicates whether to skip loading sample data into your Atlas cluster.
+   * - --tag
```

Comment:
shuold it be set here? not sure it's supported for mongocli

Reply:
this is still an atlas command (mongocli atlas quickstart).
From what I see we're keeping the docs in sync, for example: https://github.com/mongodb/mongodb-atlas-cli/blob/5d4dd4bf038f41d7496e353f4a6083c0aa74e51b/docs/mongocli/command/mongocli-atlas-clusters-create.txt#L111

Reply:
that is a bit of an unfortunate situation we want to remove atlas from mongocli entirely, those shared commands are a pain currently.

The deprecation message says we would stop supporting atlas in mongocli by April https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/cli/atlas/atlas_linux.go#L19